test/SpokePoolClient.ValidateFill.ts

```diff
@@ -563,19 +622,18 @@ describe("SpokePoolClient: Fill Validation", function () {
   });
 
   it("Ignores fills with deposit ID > latest deposit ID in spoke pool", async function () {
-    const sampleDeposit = await buildDeposit(hubPoolClient, spokePool_1, erc20_1, depositor, destinationChainId);
-    // Override the deposit ID that we are "filling" to be > 1, the latest deposit ID in spoke pool 1.
-    await buildFill(
-      spokePool_2,
-      erc20_2,
+    const deposit = await depositV3(
+      spokePool_1,
+      destinationChainId,
       depositor,
-      relayer,
-      {
-        ...sampleDeposit,
-        depositId: 2,
-      },
-      1
+      inputToken,
+      inputAmount,
+      outputToken,
+      outputAmount
     );
+
+    // Override the deposit ID that we are "filling" to be > 1, the latest deposit ID in spoke pool 1.
+    await fillV3Relay(spokePool_2, { ...deposit, depositId: deposit.depositId + 1 }, relayer);
```

Comment:
QQ: We had `depositId: 2` in the pre-diff code. Should we maintain that or is `deposit.depositId + 1` good enough

Reply:
The intent of the code was just to create a mismatched fill. I think the updated implementation is preferable because it doesn't hardcode the depositId, so it's more reflexive if the surrounding code changes.